internal/work/dispatcher.go

```diff
@@ -89,23 +89,28 @@ func (d *Dispatcher) Connect() error {
 
 	reply, err := d.conn.RequestName("com.redhat.Yggdrasil1.Dispatcher1", dbus.NameFlagDoNotQueue)
 	if err != nil {
-		return fmt.Errorf("cannot request name on bus: %v", err)
+		return fmt.Errorf("cannot request name com.redhat.Yggdrasil1.Dispatcher1 on bus: %v", err)
 	}
-
 	if reply != dbus.RequestNameReplyPrimaryOwner {
-		return fmt.Errorf("name already taken")
+		return fmt.Errorf("name com.redhat.Yggdrasil1.Dispatcher1 already taken")
 	}
 
 	log.Infof("exported /com/redhat/Yggdrasil1/Dispatcher1 on bus")
 
 	// Add a match signal on the
 	// org.freedesktop.DBus.Properties.PropertiesChanged signal.
 	if err := d.conn.AddMatchSignal(dbus.WithMatchPathNamespace("/com/redhat/Yggdrasil1/Worker1"), dbus.WithMatchInterface("org.freedesktop.DBus.Properties"), dbus.WithMatchMember("PropertiesChanged")); err != nil {
-		return fmt.Errorf("cannot add signal match: %v", err)
+		return fmt.Errorf(
+			"cannot add signal match on the org.freedesktop.DBus.Properties.PropertiesChanged: %v",
+			err,
+		)
 	}
 
 	if err := d.conn.AddMatchSignal(dbus.WithMatchPathNamespace("/com/redhat/Yggdrasil1/Worker1"), dbus.WithMatchInterface("com.redhat.Yggdrasil1.Worker1"), dbus.WithMatchMember("Event")); err != nil {
-		return fmt.Errorf("cannot add signal match: %v", err)
+		return fmt.Errorf(
+			"cannot add signal match on the com.redhat.Yggdrasil1.Worker1.Event: %v",
```

Comment:
Minor grammatical request.
```suggestion
			"cannot add signal match on com.redhat.Yggdrasil1.Worker1.Event: %v",
```